plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -671,4 +674,181 @@ private function normalize_raw_meta_data( array $data ) : array {
 		return $clubbed_data;
 	}
 
+	/**
+	 * Set custom order tables (HPOS) to authoritative if:
+	 * 1. HPOS and posts tables are in sync, or,
+	 * 2. This is a new shop (in this case also create tables).
+	 * and
+	 * 3. All installed WC plugins are compatible.
```

Comment:
I think this needs to be all on one line, otherwise we only get the first line when we run either of `wp help wc cot` or `wp help wc cot enable`:

<div align="center"><img width="700" alt="wp-wc-help" src="https://github.com/woocommerce/woocommerce/assets/3594411/578870fe-6090-40e7-934b-93f14eb0770f"></div>

Perhaps we could do:

```suggestion
	 * Set custom order tables (HPOS) to authoritative if 1) HPOS and posts tables are in sync, or, 2) This is a new shop (in this case also create tables) and, 3) All installed WC plugins are compatible.
```

Reply:
Addressed in https://github.com/woocommerce/woocommerce/pull/39865/commits/073042b52a47dae8ee3f90f3e3fc22e7a4666a17